src/components/Drawer/Drawer.tsx

```diff
@@ -20,16 +20,32 @@ export interface DrawerProps extends React.HTMLAttributes<HTMLDivElement> {
    * The content of the drawer
    */
   children?: React.ReactNode;
+  /**
+   * Drawer optional variant
+   */
+  variant?: 'bottomSheet' | 'default';
```

Comment:
could we change this `variant` to just be `drawerOpenSide`. Because a `variant` of `bottomSheet` will ignore the `drawerOpenSide` Perhaps since we're thinking left right and bottom `position` makes more sense than `drawerOpenSide`?